clients/src/test/java/org/apache/kafka/common/security/oauthbearer/internals/secured/RefreshingHttpsJwksTest.java

```diff
@@ -134,6 +136,20 @@ public void testSecondaryRefreshAfterElapsedDelay() throws Exception {
         String keyId = "abc123";
         MockTime time = new MockTime();
         HttpsJwks httpsJwks = spyHttpsJwks();
+
+        try (RefreshingHttpsJwks refreshingHttpsJwks = getRefreshingHttpsJwks(time, httpsJwks, mockExecutorService(time))) {
+            refreshingHttpsJwks.init();
+            // We refresh once at the initialization time from getJsonWebKeys.
+            verify(httpsJwks, times(1)).refresh();
+            assertTrue(refreshingHttpsJwks.maybeExpediteRefresh(keyId));
+            verify(httpsJwks, times(2)).refresh();
+            time.sleep(REFRESH_MS + 1);
```

Comment:
Why were we using real time here at all 😅 
So now we refresh once more due to mock time passing?

Reply:
> Why were we using real time here at all 😅

This test has been using `MockTime` since the start, though 😕

Reply:
this code was not changed. I just inserted the new `mockExecutorService` method in a way that generated a weird diff.

Reply:
Ah -- I saw Time time removed and assumed we were using real time.